datasets/eu/esma_sanctions/eu_esma_sanctions.yaml

```diff
@@ -0,0 +1,25 @@
+title: "EU ESMA Sanctions"
+entry_point: crawler.py
+prefix: esma-sanctions
+coverage:
+  frequency: daily
+  start: 2024-03-12
+summary: |
+  List of sanctioned entities, as published by the European Securities and Markets Authority (ESMA).
+  The directives and regulations underpinning the sanctions can be found in the [Interactive Single Rulebook.](https://www.esma.europa.eu/publications-and-data/interactive-single-)rulebook
```

Comment:
I think the summary field doesn't have markdown. This may want to move down to description.